src/modules/sandbox/components/Argument/PrimitiveArgument.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+
+import { ClarityAbiType, getTypeString } from '@stacks/transactions';
+import { Box, Input, color } from '@stacks/ui';
+
+import { Caption, Text } from '@components/typography';
+
+import { CommonArgumentProps } from './types';
+
+export const PrimitiveArgument: FC<
```

Comment:
nit: a name like 'PrimitiveArgumentInput' would make it clearer what this component is